### ISSUE TITLE
Legacy and new BigQuery drivers have the same name

### ISSUE THREAD

**Issue by akiselev** [metabase/metabase#18158]
**Describe the bug**
When adding a new database, the `Database type` dropdown shows two different BigQuery drivers with the same name:

![image](https://user-images.githubusercontent.com/653604/135505068-b62b0051-3d89-4f2d-b8a1-9ebdaffd5adf.png)

Two weeks ago during the previous bug bash, this was not the case and it was obvious which was the new driver. Now, the only way to tell the difference is the new `Project ID` override:

![image](https://user-images.githubusercontent.com/653604/135505946-eac2339d-0f37-473b-b24d-da636edf3d8b.png)

**To Reproduce**
1. Go to `Settings` > `Admin`
2. Click on `Databases` in the nav bar at the top
3. Click `Add Database`
4. Open `Database Type` dropdown

**Expected behavior**
There should be some way to tell the difference (the legacy driver should be named `BigQuery (Legacy)` or something)

**Information about your Metabase Installation:**
tag `v0.41.0-RC1` sha `6ff4eda63`

**Comment by jeff303** [comment]
@akiselev , are you running a local build or the uberjar/Docker image?  This can happen if the drivers weren't rebuilt (which I faced quite a lot when doing development, switching between branches, and whatnot).

I just tried the Docker image `metabase/metabase:v0.41.0-RC1` and do not see this problem.

<img width="811" alt="Screen Shot 2021-09-30 at 1 03 04 PM" src="https://user-images.githubusercontent.com/3521562/135507542-e8ef8867-efa9-4888-b7b4-9a270937eec8.png">
<img width="811" alt="Screen Shot 2021-09-30 at 1 02 58 PM" src="https://user-images.githubusercontent.com/3521562/135507543-74a8d3ba-cbb6-497d-b120-5ba2c7b395c7.png">
<img width="727" alt="Screen Shot 2021-09-30 at 1 02 52 PM" src="https://user-images.githubusercontent.com/3521562/135507546-db33f737-590c-4705-8040-86469a9c4324.png">
